config/apPipe.py

```diff
@@ -14,6 +14,7 @@
 config.ccdProcessor.calibrate.load(os.path.join(configDir, 'calibrate.py'))
 
 # Use dataset's specific templates
+config.retrieveTemplate.load(os.path.join(configDir, 'retrieveTemplate.py'))
```

Comment:
The use of Python config files (in fact, all of `apPipe.py`) was a workaround to simultaneously support Gen 2 and Gen 3. I recommend leaving this package unchanged.